player/esbuild.mjs

```diff
@@ -10,35 +11,21 @@ if (!existsSync(buildDir)) {
   mkdirSync(buildDir)
 }
 
-const bundles = [
-  { format: 'esm', name: 'index-esm.js' },
-  { format: 'cjs', name: 'index-cjs.js' },
-]
-
-for (const { name, format } of bundles) {
-  buildSync({
-    platform: 'browser',
-    entryPoints: ['src/index.ts'],
-    outfile: join(buildDir, name),
-    format,
-    external: [
-      '@juggle/resize-observer',
-      'debug',
-      'react-hooks-shareable',
-      'react',
-      'react-dom',
-      'luxon',
-      'styled-components',
-      'media-stream-library',
-    ],
-    bundle: true,
-    minify: false,
-    sourcemap: true,
-    // avoid a list of browser targets by setting a common baseline ES level
-    target: 'es2015',
-  })
-}
+// ES module
+buildSync({
+  platform: 'browser',
+  entryPoints: ['src/index.ts'],
+  outfile: join(buildDir, 'index.js'),
+  format: 'esm',
+  packages: 'external',
+  bundle: true,
+  minify: false,
+  sourcemap: true,
+  // avoid a list of browser targets by setting a common baseline ES level
+  target: 'es2015',
```

Comment:
Just an observation, probably not relevant for you. In the latest version of esbuild, 0.24.2, you can use "es2024" as target.